draft-pauly-privacypass-auth-scheme.md

```diff
@@ -304,4 +380,24 @@ Pointer to specification text: {{challenge-redemption}} of this document
 
 ## Token Type Registry {#token-types}
 
-TODO: Create new registry for token types
+The "Token Type" registry lists identifiers for issuance protocols defined for use
+with the Privacy Pass token authentication scheme. These identifiers are two-byte values,
+so the maximum possible value is 0xFFFF = 65535.
+
+Template:
+
+* Value: The two-byte identifier for the algorithm
+* Name: Name of the issuance protocol
+* Public: A Y/N value indicating if the output tokens are publicly verifiable
+* Nk: The length in bytes of an output authenticator
+* Reference: Where this algorithm is defined
+
+The initial contents for this registry are defined in the table below.
+
+| Value  | Name                   | Public | Nk  | Reference    |
+|:-------|:-----------------------|:-------|:----|:-------------|
+| 0x0000 | (reserved)             | N/A    | N/A | N/A          |
+| 0x0001 | Blind RSA, 2048        | Y      | 256 | TODO         |
+| 0x0002 | Blind RSA, 4096        | Y      | 512 | TODO         |
+| 0x0003 | VOPRF(P-384, SHA-384)  | N      | 256 | TODO         |
```

Comment:
So PATs would be another entry in this table?

I wonder if the name should be a bit more descriptive than the signature type. Or I guess PATs would be something like "Rate-Limited Blind RSA"?

Reply:
Yeah, these are sort of placeholder names until we have the specific drafts to point to. "Blind RSA" isn't very helpful, for example.

Reply:
Wonder if we should leave some space so that we can at least keep similar entries close together numerically?

Though I guess that devolves into a type/version separation again.

Reply:
Yeah, numbers should be cheap... but we can add in batches.

We can fill this out as we write the three issuance protocol specs

Reply:
Yeah -- let's just fill this stuff out later.